head/core/tasks_proto.cpp

```diff
@@ -82,16 +76,11 @@ void head_tasks::start_tasks(
     auto& spi3_task = spi3_task_builder.start(5, "spi bus 3", spi3_device);
     spi3_task_client.set_queue(&spi3_task.get_queue());
 
-    auto& presence_sensing = presence_sensing_driver_task_builder.start(
```

Comment:
shouldn't we keep presence sensing on pre-EVT head boards if it's not too much of a pain?

Sidenote, I feel like we're getting pretty close to where we can delete proto code for a bunch of stuff.

Reply:
Well, maybe... but I'm more inclined to agree with the second statement (for a different PR since this one is so time-constrained)

Reply:
Also presence sensing will have to look really different anyway